src/mixins/conversation.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @copyright Copyright (c) 2019
```

Comment:
:eyes: 

Let's do it a util instead. We're trying to get rid of mixins in projects, iirc
Besides, if it's not going to be used in Dashboard, we could just keep it as LeftSidebar method

Reply:
I thought about making it independent from LeftSidebar so we can have quick methods to check the mention flag and the unread flag in the future without retyping all of those conditions.

Reply:
Then also could be a util, or composable.